styles.css

```
@@ -0,0 +1,62 @@
+
+body {
+    font-family: Arial, sans-serif;
+    line-height: 1.6;
+    background-color: black;
+    color: white;
+    padding: 20px;
+}
+
+.contenedor {
+    display: flex;
+    justify-content: space-between;
+    gap: 20px;
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+.barrita {
+    flex: 1;
+    background-color: wheat;
+    padding: 20px;
+    border-radius: 8px;
+}
+
+.impoconte {
+    flex: 2;
+    background-color:orange;
+    padding: 20px;
+    border-radius: 8px;
+}
+
+h1, h2 {
+    color: white;
+}
+
+ul {
+    list-style-type: none;
+    margin-top: 10px;
+}
+
+li {
+    margin: 8px 0;
+}
+
+footer {
+    background-color: purple;
+    color: white;
+    text-align: center;
+    padding: 15px 0;
+    margin-top: 30px;
+    border-radius: 8px;
+}
+
+footer p {
+    font-size: 14px;
+}

```

```suggestion
footer {
    background-color: purple;
    color: white;
    text-align: center;
    padding: 15px 0;
    margin-top: 30px;
    border-radius: 8px;
    
    p {
        font-size: 14px;
    }
}
```

Puede usar CSS Nesting